### PR TITLE
Feature/better choose menu

### PR DESCRIPTION
**Pull request by mkfink**
Adds query option to query_tas_status() and flash.py to display to display current device status (software versions, state) in the device selection menu when flashing. Defaults to false so current behavior should not change. Have tested.

### CODE REVIEW

File: espq.py [hunk 1]
```diff
@@ -481,11 +485,28 @@ def choose_devices(devices):
 
     choice_list = []
     current_type = None
+    name_len_spacer = max([len(dev.name) for dev in devices]) + 1
+
+    if query == True:
+        print('Querying all devices for version numbers. This make take a minute...')
```
Comment (mtfurlan) [inline espq.py]: "this make take a minute"

File: espq.py [hunk 2]
```diff
@@ -481,11 +485,28 @@ def choose_devices(devices):
...
+        choice_list.append(Separator('Name'.center(name_len_spacer) + 'Firmware'.center(15) + 'Core'.center(7) + 'State'))
+
     for device in devices:
```
Comment (mtfurlan) [inline espq.py:494]: This expects devices to be sorted.
Should we sort it here?

**Review by mtfurlan — APPROVED** [review]
I put two comments in, fix the typo and it's fine.

File: espq.py (hunk 2 above)
Comment (mkfink) [reply espq.py:494]: They're already sorted in import_devices()

**Pull request merged**